ytdlp_handler.py

```diff
@@ -39,11 +40,28 @@ def video_dl(values: Dict) -> None:
     with yt_dlp.YoutubeDL(ydl_opts) as ydl:
         infos_ydl = ydl.extract_info(values["url"])
     DL_PROGRESS_WINDOW.close()
+
+    if infos_ydl["_type"] == "playlist":
```

Comment:
I thinks it's not a good idea to have a tick without other options (like a begin and start video in GUI according to your remarks on twitter [here](https://twitter.com/Kenshin9977/status/1477398559225958408?s=20)). 
Sure, this can be done in the future when it's necessary. For now, I think it's more interesting for the user just to have copied a link (video or playlist) and download. He probably already knows what he wants to download because he manually copy the link.

But you can decide, I can add this tick without other options if you really want it now ?